Casks/c/conar.rb

```diff
@@ -1,19 +1,27 @@
 cask "conar" do
   arch arm: "arm64", intel: "x64"
 
-  version "0.25.0"
-  sha256 arm:   "ea69b16ac16ea55dd85fe5aaf29d4872dbf84d42d69d15ea56d4270453fe2432",
-         intel: "4c6b98e0d09684eedd787fe09c41d1d713fffdfc1b2872873c73c3cd0a341af0"
+  version "0.25.4,251215sjkp8bkh6"
+  sha256 arm:   "b7fd6c52780759d45ed2887087bc9489d1db30f46602e704f1190d0fd3c93d3d",
+         intel: "52b100d0066fdbb782a47a61972ec511519643be604b3dd09940918a1b275a91"
 
-  url "https://github.com/wannabespace/conar/releases/download/v#{version}/Conar-Mac-#{arch}-#{version}-Installer.dmg",
-      verified: "github.com/wannabespace/conar/"
+  url "https://download.todesktop.com/25112796k32u7/Conar%20#{version.csv.first}%20-%20Build%20#{version.csv.second}-#{arch}-mac.zip",
+      verified: "download.todesktop.com/25112796k32u7/"
   name "Conar"
   desc "AI-powered database and data management tool"
   homepage "https://conar.app/"
 
   livecheck do
-    url :url
-    strategy :github_latest
+    url "https://download.todesktop.com/25112796k32u7/latest-mac.yml"
+    regex(/Conar\sv?(\d+(?:\.\d+)+)\s-\sBuild\s([a-z\d]+)-#{arch}-mac\.zip/i)
```

Comment:
```suggestion
    regex(/Conar\s*v?(\d+(?:\.\d+)+)\s*-?\s*(?:Build\s*([a-z\d]+?)[._-])?#{arch}[._-]mac\.zip/i)
```

This seems to borrow from existing `livecheck` blocks (e.g., `whimsical` and/or `tandem`) but, unlike those other casks, the second capture group isn't optional in this one (so the related `match[2].present?` `strategy` block logic doesn't make sense). The unversioned download link's `Content-Disposition` header uses a file name like "Conar 0.25.4 - x64.dmg", so we should arguably adapt this regex to be able to match that format as well. That's kind of what the aforementioned casks are doing with the optional second capture group but not quite in the same way (I'll see about aligning them in a follow-up PR).

Besides that, this suggestion provides a bit more flexibility in the regex (e.g., `\s*` instead of strictly matching one whitespace character with `\s`, `[._-]` for suffix delimiters, etc.).